vs/README.md

```diff
@@ -0,0 +1,200 @@
+                     "--model model/model.npz",
+                     "--train-sets data/corpus.bpe.ro data/corpus.bpe.en",
+                     "--vocabs model/vocab.ro.yml model/vocab.en.yml",
+                     "--dim-vocabs 66000 50000",
+                     "--mini-batch-fit",
+                     "-w 3000",
+                     "--layer-normalization",
+                     "--dropout-rnn 0.2",
+                     "--dropout-src 0.1",
+                     "--dropout-trg 0.1",
+                     "--early-stopping 5",
+                     "--valid-freq 100",
+                     "--save-freq 10000",
+                     "--disp-freq 100",
+                     "--valid-metrics cross-entropy translation",
+                     "--valid-sets data/newsdev2016.bpe.ro data/newsdev2016.bpe.en",
+                     "--valid-script-path .\\scripts\\validate.bat",
+                     "--log model/train.log",
+                     "--valid-log model/valid.log",
+                     "--overwrite",
+                     "--keep-best",
+                     "--seed 1111",
+                     "--exponential-smoothing",
+                     "--normalize=1",
+                     "--beam-size=12",
+                     "--quiet-translation"
+                 ]
+                 }
+             ]
+         }
+
+
+
+### 2. Create solution and projects files for Visual Studio : `CreateVSProjects.bat`
+
+If you have a previous version of Visual Studio, you will need to use CMake to generate the projects files.
+
+The provided script `CreateVSProjects.bat` runs the dependency checks then invokes CMake with the right parameters to create the solutions for Visual Studio.
+
+
+### 3. Use MSBuild : `Build.bat`
+
+The last alternative is to use the script `Build.bat` that will:
+- Check the dependencies
+- Create the VS project files
+- Invoke MSBuild on these projects to build the targets in Release.
+
+---
+## Changes from the master branch
+This part gives more information on all changes done. Refer to [this page](https://github.com/cedrou/marian-dev/commits/build_on_win) for commits.
+
+1. __Fix Cuda error : Unsupported Visual Studio Version Error__   
+   See above for justification and fixes
+
+2. __Fix VS compiler flags / Build in Release, with improved debug info__  
+   Added VS specific compile and link flags
+
+3. __Fix Warning: D9002: ignoring unknown option '-m64'__  
+   This one is related to a compiler flag added while finding the package MKL that does not exists for MS compiler. 
+
+4. __Fix marian::Backend, marian::cpu::Backend and marian::gpu::Backend conflicts__  
+   There were name conflicts between the 3 `Backend` classes that confused the compiler:
+   
+   >  template instantiation resulted in unexpected function type of "type" (the meaning of a name may have changed since the template declaration -- the type of the template is "type").
+
+   I renamed the CPU and GPU as `cpuBackend` and `gpuBackend`.

```

Comment:
Is this still accurate, or was this rolled back? Would you mind quickly going over these once again to check whether some of these changes were undone or changed further since you wrote this? Then please close this Conversation.